### DISCUSSION TITLE
Checkbox doesn't stay checked in the Mandatory Message

### DISCUSSION THREAD

**Discussion by maestro1315** [community/community#39774] [Enterprise] [answered]
### Select Topic Area

Bug

### Body

Checkmark disappears after the box is checked and saved is clicked. 
"Show updated message to all users even if they dismissed the previous one."

**Comment by ossanna16** [comment, marked as answer]
Hi there @maestro1315 and welcome to our community! Thank you for reporting this bug. Can you please add more detailed information so we can try to help?

**Reply by maestro1315** [reply]
https://docs.github.com/en/enterprise-server@3.7/admin/user-management/managing-users-in-your-enterprise/customizing-user-messages-for-your-enterprise - Step #6 of Creating a Mandatory Message. The checkbox will not stay checked.

**Reply by maestro1315** [reply]
Confirmed this is still not working in 3.5.7.
It appears that even though the checkbox for mandatory messages will not stay checked the mandatory message will present itself. It will however only present itself for built-in users, if the users are provisioned using SAML/SCIM they will not see the mandatory message when logging in.
